site/gatsby-site/src/components/Layout.js

```diff
@@ -24,7 +24,7 @@ const Layout = ({ children, className, sidebarCollapsed = false, location }) =>
         <div
           id="content"
           className={
-            'flex flex-grow pt-4 px-4 pb-5 md:px-10 md:pb-10 z-[1] relative flex-1 overflow-clip' +
+            'flex flex-grow pt-4 px-4 pb-5 md:px-10 md:pb-10 z-1 relative flex-1 overflow-clip' +
```

Comment:
Does z-1 actually do anything? It seems like [Tailwind jumps from z-0 to z-10 for whatever reason](https://tailwindcss.com/docs/z-index).